.pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+gulp unit-test
```

Comment:
should we add jshint to pre-commit hook to prevent committing any unlinted code?